contracts/smart-account/modules/EcdsaOwnershipRegistryModule.sol

```diff
@@ -106,6 +107,43 @@ contract EcdsaOwnershipRegistryModule is
         bytes memory moduleSignature,
         address smartAccount
     ) public view virtual override returns (bytes4) {
+        if (
+            _verifySignature(
+                keccak256(
+                    abi.encodePacked(
+                        "\x19Ethereum Signed Message:\n52",
+                        dataHash,
+                        smartAccount
+                    )
+                ),
+                moduleSignature,
+                smartAccount
+            )
+        ) {
+            return EIP1271_MAGIC_VALUE;
+        }
+        return bytes4(0xffffffff);
+    }
+
+    /// @inheritdoc IEcdsaOwnershipRegistryModule
+    function isValidSignatureUnsafe(
+        bytes32 dataHash,
+        bytes memory moduleSignature
+    ) public view virtual returns (bytes4) {
+        return
+            isValidSignatureForAddressUnsafe(
+                dataHash,
+                moduleSignature,
+                msg.sender
+            );
+    }
+
+    /// @inheritdoc IEcdsaOwnershipRegistryModule
+    function isValidSignatureForAddressUnsafe(
```

Comment:
same here, what do you think about it?

Reply:
I don't mind using a less alarming name. Which one would you suggest?

((My initial thought to use this name was to highlight this can be not safe so protocols double check they know what they do when using this module))

Reply:
Let's keep it like this for now, let's add a visible warning to NatSpec in the interface. We can discuss it with the team tomorrow. Maybe I'm the only one who is concerned 🥸

Reply:
I like this name :P, we could go for something even more alarming like https://blog.logrocket.com/using-dangerouslysetinnerhtml-in-a-react-application/